server/core/src/main/java/org/infinispan/server/core/telemetry/TelemetryServiceFactory.java

```diff
@@ -28,6 +28,10 @@ public Object construct(String name) {
       }
 
       try {
+         // At the moment we don't export Infinispan server metrics with OpenTelemetry,
+         // so we manually disable any metrics export
+         System.setProperty("otel.metrics.exporter", "none");
```

Comment:
I don't know. I'm going to explore the way the Quarkus OpenTelemetry extension does the same, or we can figured out something else.

Reply:
The Quarkus OpenTelemetry extension does not use the Autoconfigure SDK module, so they build the OpenTelemetry SDK on their own.

The alternative would be to provide a do-nothing `MetricExporter`.
For instance:

``` java
openTelemetry = AutoConfiguredOpenTelemetrySdk.builder()
            .addMetricExporterCustomizer((metricExporter, configProperties) -> new NoExportMetricExporter())
            .build()
            .getOpenTelemetrySdk();
```

and

```
private static class NoExportMetricExporter implements MetricExporter {

  @Override
  public CompletableResultCode export(Collection<MetricData> metrics) {
     return CompletableResultCode.ofSuccess();
  }

  @Override
  public CompletableResultCode flush() {
     return CompletableResultCode.ofSuccess();
  }

  @Override
  public CompletableResultCode shutdown() {
     return CompletableResultCode.ofSuccess();
  }

  @Override
  public AggregationTemporality getAggregationTemporality(InstrumentType instrumentType) {
     return AggregationTemporality.CUMULATIVE;
  }
}
```

but I don't think it is better, it looks like something that consumes resources.

I have no other ideas.
Alternatively we could avoid using the Autoconfigure SDK module, but it would be a shame, because this module allows us to configure a lot of stuff (e.g. batches) and we should reimplement all by ourselves.

Of course any further ideas are more than welcome, as usual.